datadog/internal/validators/validators.go

```diff
@@ -52,6 +52,15 @@ func ValidateEnumValue(newEnumFunc interface{}) schema.SchemaValidateFunc {
 	}
 }
 
+// ValidateStringValue ensures that a string value is valid
+func ValidateStringValue(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
Could we use `StringIsNotEmpty` from the default package instead?

Reply:
Done in [commit 2f914c9](https://github.com/DataDog/terraform-provider-datadog/pull/892/commits/2f914c9da8e833c9a46c5c24bd654aa47ba44918)